pages/learn/develop/local-mode.md

```diff
@@ -122,14 +122,14 @@ These containers will have access to all the features and environment that balen
 
 ## SSH into the running app container or host OS
 
-If we want to run some test commands in our app container, we can do this easily using `{{ $names.company.short }} local ssh`. This command drops us directly into the selected container:
+If we want to run some test commands in our app container, we can do this easily using `{{ $names.company.short }} ssh`. This command drops us directly into the hostOS. From here, we can check system logs and [perform other troubleshooting tasks][troubleshooting]:
 ```
-sudo {{ $names.company.short }} local ssh f340127.local
+sudo {{ $names.company.short }} ssh f340127.local
```

Comment:
While at it, would it be preferably to use an IP address (like `192.168.1.10`) instead of a `.local` domain name? While `.local` domain names are nice and usually work, the problem is the adverb "usually". I've replied to a few support conversations in the past whether users couldn't get avahi / local domain resolution to work in their local networks. I myself couldn't get it to work when I used my MacBook's "internet connection sharing" to provide wired Ethernet access to an RPi 2 that didn't have built-in WiFi. For a "learn" / "get started" doc, arguably we would want to priority certainty / reliability over cool features that usually work...

Reply:
That seems reasonable to me, will update that.